preview.yml

```diff
@@ -14,7 +14,7 @@ content:
 
 ui:
   bundle:
-    url: https://s3-eu-west-1.amazonaws.com/static-content.neo4j.com/build/ui-bundle-latest.zip
+    url: https://d12wh7zj8x3amw.cloudfront.net/build/ui-bundle-latest.zip
```

Comment:
Do we consider this relatively stable now?

Reply:
Idk, it was the link that Adam sent yesterday

Reply:
If you don't mind the possibility of updating again later, it's fine to use this. I can't say 100% that we won't offer a different bundle URL soon, but at least this one works

Reply:
Yep it's ok, I'm still working on other things in this repo anyway!

Reply:
Sounds good.